### PR TITLE
Add job for building machine-api-operator image

### PR DESCRIPTION
**Pull request by mkumatag**
Job for building `machine-api-operator` image

### CODE REVIEW

File: config/jobs/periodic/machine-api-operator/build-machine-api-operator-image.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                  namespace: ${namespace}
+                  user: default-user
+              current-context: default-context
+              users:
+              - name: default-user
+                user:
+                  token: ${token}
+              EOL
+              export KUBECONFIG=$(pwd)/kubeconfig
+              kubectl build --push --registry-secret quay-powercloud-regcred --namespace image-builder -t quay.io/powercloud/machine-api-operator:release-4.8 -t quay.io/powercloud/machine-api-operator:latest -f Dockerfile.ppc64le .
```
Comment (Rajalakshmi-Girish) [inline config/jobs/periodic/machine-api-operator/build-machine-api-operator-image.yaml:67]: @mkumatag Don't we want to do image building on `k8s-ppc64le-cluster`?

**Review by Rajalakshmi-Girish — CHANGES_REQUESTED** [review]

File: config/jobs/periodic/machine-api-operator/build-machine-api-operator-image.yaml (hunk 1 above)
Comment (mkumatag) [reply config/jobs/periodic/machine-api-operator/build-machine-api-operator-image.yaml:67]: not really, I'm cross-building the binary on the x86 platform itself.

**Comment by mkumatag** [comment]
Currently, these changes are just building things for ppc64le platform, I will have to work on building these images on x86 as well and push fat manifest.

**Comment by Rajalakshmi-Girish** [comment]
> 
> 
> Currently, these changes are just building things for ppc64le platform, I will have to work on building these images on x86 as well and push fat manifest.

Do you mean someone who is working from `x86` and would like to have `machine-api-operator` for/on power can use this image?
Apologies if I am framing the question wrongly :)

**Comment by mkumatag** [comment]
> > 
> > 
> > Currently, these changes are just building things for ppc64le platform, I will have to work on building these images on x86 as well and push fat manifest.
> 
> Do you mean someone who is working from `x86` and would like to have `machine-api-operator` for/on power can use this image?
> Apologies if I am framing the question wrongly :)
> 
> 

It's like someone can trigger this code from x86 box as well..

**Comment by Rajalakshmi-Girish** [comment]
> 
> 
> > > Currently, these changes are just building things for ppc64le platform, I will have to work on building these images on x86 as well and push fat manifest.
> > 
> > 
> > Do you mean someone who is working from `x86` and would like to have `machine-api-operator` for/on power can use this image?
> > Apologies if I am framing the question wrongly :)
> 
> It's like someone can trigger this code from x86 box as well..

Infact ONLY from `x86` box as we have image only built on `iks prow cluster` and not on `ppc64le`.

**Pull request merged**